app/models.py

```diff
@@ -2,6 +2,7 @@
 from datetime import datetime
 from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy.orm import relationship, backref
+from sqlalchemy import func
```

Comment:
IMPORT THE FUNK 🤠🥳
![funky gif](https://media.giphy.com/media/3oEjHQKtDXpeGN9rW0/giphy.gif)

Reply:
Amazing!